daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -236,7 +236,7 @@ private[lf] final class Compiler(
       ref: SDefRef,
       arity: Int,
   )(
-      body: List[Position] => SExpr
+      body: PartialFunction[List[Position], SExpr]
```

Comment:
mostly to shut up the exhaustiveness checker. We really don’t want to write all those functions exhaustively.